downstairs/src/lib.rs

```diff
@@ -1686,25 +1681,62 @@ where
      * gracefully.  By exiting the loop here, we allow the calling
      * function to take over and handle a reconnect or a new upstairs
      * takeover.
+     *
+     * Tasks are organized as follows, with tasks in `snake_case`
```

Comment:
Oh perfect.

I thought going in to this review that my only comment was going to be to add the
picture from the PR to the code somewhere, but you have already done that.